### PR TITLE
Update Cloud to accept log messages from V2X Hub (SPaT) - Binary

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
As an ODE administrator, I want to accept and decode log messages from V2X Hub and publish on Kafka topics available for cloud or database consumers. These logs from the V2X Hub may come to the ODE via batch upload logs or Kafka topics.

## Related Issue
https://github.com/usdot-jpo-ode/jpo-ode/issues/420

## Motivation and Context
ODE and V2xHub integration

## How Has This Been Tested?
Local Integration test

## Types of changes
N/A

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[ODE Contributing Guide](https://github.com/usdot-jpo-ode/jpo-ode/blob/bugfix/Pull_request_template/docs/contributing_guide.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dan-du-car** [comment]
After upload binary file to bsmlog folder, the  console below shows that binary file is successfully uploaded and parsed.
image

Below image shows listening to the topic.Asn1DecodeOutput topic to view the decoded xml messages:
image

**Comment by dan-du-car** [comment]
Upload a sample spat binary file, the file is parsed successfully and moved to backup folder.
image

Codec service is able to decode the upload file and send the result to Kafka topic: 
image

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/IntersectionParser.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	}
+
+	public void setIntersection(LogIntersection intersection) {
+		this.intersection = intersection;
+	}
+	 public void setIntersection(byte code) {
+	      try {
+	    	  this.intersection.setIntersectionStatus(code);
+	      } catch (Exception e) {
+	         logger.error("Farse Intersection status value error, Invalid value {}: ",
```
Comment (TonyEnglish) [inline jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/IntersectionParser.java:70]: Is this supposed to be False?

**Review by TonyEnglish — COMMENTED** [review]
Added one comment, overall looks really good!

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/IntersectionParser.java (hunk 1 above)
Comment (dan-du-car) [reply jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/parser/IntersectionParser.java]: It is updated. https://github.com/usdot-jpo-ode/jpo-ode/pull/419/commits/672bca0a59705c93eea70553f1edfee0dda04a48

**Review by TonyEnglish — APPROVED** [review]

**Pull request merged**